dnsmasq/apparmor.txt

```diff
@@ -25,11 +25,15 @@ profile dnsmasq flags=(attach_disconnected,mediate_deleted) {
   /dev/tty rw,
   /tmp/* rw,
 
+  # S6-Overlay
+  /bin/** ix,
```

Comment:
Need a bit more cleanp. Like `/bin/busybox ix,` would be covered also with `/bin/** ix` ect